src/index.html

```diff
@@ -189,7 +192,12 @@
       <div class="footer-bar">
         &nbsp;
         <div class="footer-bar-content">
-          <a href="/">HOME</a>
+            <a href="/" class="link">HISTORY</a> 
+            <a href="/" class="link">PEOPLE</a> 
+            <a href="/" class="link">INSTITUTIONS</a> 
+            <a href="/" class="link">ACTIVITIES</a> 
+            <a href="/" class="link">PUBLICATIONS</a> 
+            <a href="/" class="link">CONTACT / IMPRINT</a>
```

Comment:
Right now this looks like: 
![image](https://user-images.githubusercontent.com/49606945/76199891-f5aca880-61f0-11ea-95ea-6ba92c989042.png)
So it needs some styling because it just looks like one line. And it also needs routing to these pages.